python_modules/dagster/dagster/_core/definitions/asset_graph_subset.py

```diff
@@ -314,7 +322,7 @@ def from_asset_keys(
                     asset_key
                 ] = partitions_def.empty_subset().with_partition_keys(
                     partitions_def.get_partition_keys(
-                        dynamic_partitions_store=dynamic_partitions_store
+                        dynamic_partitions_store=dynamic_partitions_store, current_time=current_time
```

Comment:
discovered a bug where `current_time` isn't being passed into the `AssetGraphSubset.all` selection